Controllers/SubscriptionsController.cs

```diff
@@ -101,5 +101,26 @@ public async Task<Subscription> RemoveSubscription(string userId, [FromBody] Sub
 
             return subscription;
         }
+
+        /// <summary>
+        /// Remove a subscription
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <param name="subscription"></param>
+        /// <returns></returns>
+        [HttpDelete]
+        [Route("{userId}/sub/removeAll")]
```

Comment:
```suggestion
        [Route("{userId}/sub/all")]
```
the `removing` is indicated by the `HttpDelete`